src/server/editorServices.ts

```diff
@@ -2489,7 +2489,7 @@ namespace ts.server {
         }
 
         private getOrCreateScriptInfoNotOpenedByClientForNormalizedPath(fileName: NormalizedPath, currentDirectory: string, scriptKind: ScriptKind | undefined, hasMixedContent: boolean | undefined, hostToQueryFileExistsOn: DirectoryStructureHost | undefined) {
-            if (isRootedDiskPath(fileName) || isDynamicFileName(fileName)) {
+            if (isRootedDiskPath(fileName) || isDynamicFileName(fileName) || isUrl(fileName)) {
                 return this.getOrCreateScriptInfoWorker(fileName, currentDirectory, /*openedByClient*/ false, /*fileContent*/ undefined, scriptKind, hasMixedContent, hostToQueryFileExistsOn);

```

Comment:
i think we have to handle it in all places we check isDynamicFileName or isDynamicOrHasMixed content on script info

Reply:
Also need to handle all locations of isRootedDiskPath checks because i dont think our compiler is completely ready for that.